homeassistant/components/systemmonitor/strings.json

```diff
@@ -0,0 +1,69 @@
+{
+  "config": {
+    "error": {
+      "missing_arg": "Mandatory argument missing for requested type"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "type": "Type",
+          "arg": "Argument"
+        },
+        "data_description": {
+          "type": "The type of the information to display",
+          "arg": "Argument to use, see documentation for mandatory usage."
+        }
+      }
+    }
+  },
+  "options": {
+    "error": {
+      "missing_arg": "Mandatory argument missing for requested type"
+    },
+    "step": {
+      "init": {
+        "menu_options": {
+          "add_sensor": "Add sensor",
+          "select_edit_sensor": "Configure sensor",
+          "remove_sensor": "Remove sensor"
+        }
```

Comment:
It would be nice if we could just set up the sensors and don't have options for adding and removing. Users can still enable/disable when we have unique ids for entities.

Eg could we fetch all the available disks and network interfaces and create sensors for all of them and disable them by default? The imported options can enable the selected resources by default.

The only sensor where this could become unwieldy is the `process` sensor. If we want to keep a selector to add/edit/remove the process sensors, I'd only have it for that sensor type and not all other sensors where we don't need that.

Reply:
Ì think that could make sense. I was thinking about a middle way to make it easier but might as well just look up the network and file system and do sensors for it all disabled by default.

Reply:
I did now add it in so it should load all of it (except process) while not tested so not sure that it works (yet).

We shouldn't consider to keep add/remove etc. anyhow for all the options in case new disk mounts get's added (as example). It would restrict the user from making modifications in the case he alters his system e.g. disks, network etc?

Reply:
I think it would be better if we add new disks and networks automatically as disabled. Users can disable entities that they don't need.